lms/djangoapps/certificates/management/commands/modify_cert_template.py

```diff
@@ -0,0 +1,91 @@
+"""Management command to modify certificate templates."""
+import logging
+import shlex
+from argparse import RawDescriptionHelpFormatter
+
+from django.core.management.base import BaseCommand, CommandError
+
+from lms.djangoapps.certificates.models import (
+    ModifiedCertificateTemplateCommandConfiguration,
+)
+from lms.djangoapps.certificates.tasks import handle_modify_cert_template
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Management command to modify certificate templates.
+    Example usage:
+    ./manage.py lms modify_cert_template --old-text
+    """
+
+    help = """Modify one or more certificate templates.
+    This is DANGEROUS.
+    * This uses string replacement to modify HTML-like templates, because the presence of
+      Django Templating makes it impossible to do true parsing.
+    * This isn't parameterizing the replacement text, for the same reason.  It has
+      no way of knowing what is template language and what is HTML.
+    Do not trust that this will get the conversion right without verification,
+    and absolutely do not accepted untrusted user input for the replacement text. This is
+    to be run by trusted users only.
+    Always run this with dry-run or in a reliable test environment.
```

Comment:
Kudos 👍. Thanks for pointing out that this is a potentially dangerous and destructive action.